### ISSUE TITLE
App crashes when new files are added to a root folder

### ISSUE THREAD

**Issue by melvin4u445** [ARK-Builders/ARK-Navigator#53]
Steps:
1. Open ARK Navigator
2. Add DCIM folder as root
3. Open DCIM using the compass button
4. Now take a screenshot
5. Tap Manage Folders
6. Find DCIM and tap the compass button again

App crashes. Logs attached
[log.txt](https://github.com/kirillt/ark-navigator/files/7103617/log.txt)

**Comment by kirillt** [comment]
Thank you!

Highlight of the log:
```
09-03 08:50:45.746 23198 23198 D tags-storage: resources in the storage: [415181758, 1059249623, 1071064384, 1929120046, 3016451087]
09-03 08:50:45.747 23198 23198 D tags-storage: resources in the index: [415181758, 1059249623, 1071064384, 1929120046, 3016451087, 1457690694]
09-03 08:50:45.747 23198 23198 D AndroidRuntime: Shutting down VM
--------- beginning of crash
09-03 08:50:45.756 23198 23198 E AndroidRuntime: FATAL EXCEPTION: main
09-03 08:50:45.756 23198 23198 E AndroidRuntime: Process: space.taran.arknavigator, PID: 23198
09-03 08:50:45.756 23198 23198 E AndroidRuntime: java.lang.AssertionError: Index and storage diverged
09-03 08:50:45.756 23198 23198 E AndroidRuntime: 	at space.taran.arknavigator.mvp.model.repo.PlainTagsStorage$Companion.provide(PlainTagsStorage.kt:206)
09-03 08:50:45.756 23198 23198 E AndroidRuntime: 	at space.taran.arknavigator.mvp.presenter.ResourcesPresenter.onFirstViewAttach(ResourcesPresenter.kt:79)
```

So the index contains new id, but the storage doesn't. Right now, storage assumes to contain any id from index even if there are no tags for the id. We need to either notify storage about new ids or avoid storing non-labeled ids.

**Comment by kirillt** [comment]
Should be fixed in 82087ccd4bb391ca57566ddef308c5a032822172

**Comment by melvin4u445** [comment]
Doesn't crash anymore. Looks good 👌
